app/components/enter-hours.tsx

```diff
@@ -0,0 +1,298 @@
+"use client"
+
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card"
+import {
+    Popover,
+    PopoverContent,
+    PopoverTrigger,
+} from "@/components/ui/popover"
+import { ScrollArea } from "@radix-ui/react-scroll-area"
+import { zodResolver } from "@hookform/resolvers/zod"
+import { useForm } from "react-hook-form"
+import { Button } from "@/components/ui/button"
+import * as z from "zod"
+import {
+    Form,
+    FormControl,
+    FormField,
+    FormItem,
+    FormLabel,
+    FormMessage,
+} from "@/components/ui/form"
+import {
+    Select,
+    SelectContent,
+    SelectItem,
+    SelectTrigger,
+    SelectValue,
+} from "@/components/ui/select"
+import { useState } from "react"
+import { Calendar } from "@/components/ui/calendar"
+import { cn } from "@/lib/utils"
+import { format } from "date-fns"
+import { CalendarDays } from 'lucide-react';
+
+
+const enterHoursSchema = z.object({
+    date: z
+        .date({
+            required_error: "A date is required.",
+        }),
+    startTime: z
+        .string({
+            required_error: "We need to know when you started work to calculate your TOIL",
+        })
+        .regex(
+            /^([1-9]|1[0-2]):[0-5][0-9] (AM|PM)$/,
+            "Start time must be in the format of HH:MM AM/PM"
+        ),
+    endTime: z
+        .string({
+            required_error: "We need to know when you ended work to calculate your TOIL",
+        })
+        .regex(
+            /^([1-9]|1[0-2]):[0-5][0-9] (AM|PM)$/,
+            "End time must be in the format of HH:MM AM/PM"
+        )
+})
+
+export type EnterHours = z.infer<typeof enterHoursSchema>
+
+export default function EnterHours() {
+
+    const [hoursWorked, setHoursWorked] = useState<EnterHours[]>([])
+
+    const form = useForm<z.infer<typeof enterHoursSchema>>({
+        resolver: zodResolver(enterHoursSchema),
+        defaultValues: {
+            date: undefined,
+            startTime: "",
+            endTime: "",
+        },
+    })
+
+    function onSubmit(newHoursWorked: EnterHours) {
```

Comment:
Do something like this on top of the `onSubmit` to `reset` the form, as right now once you add it, the inputs aren't reverted.

https://github.com/troypoulter/toil-calculator/blob/20bcab4be25d8e2a2c160e11441ec566f25cc62d/app/components/rulesets.tsx#L93-L97